redisearch/client.go

```diff
@@ -499,17 +499,26 @@ func (info *IndexInfo) loadSchema(values []interface{}, options []string) {
 
 		f := Field{Name: spec[0]}
 		switch strings.ToUpper(spec[2]) {
+		case "TAG":
+			f.Type = TagField
+			tfOptions := TagFieldOptions{}
+			if wIdx := sliceIndex(options, "SEPARATOR"); wIdx != -1 {
+				tfOptions.Separator = options[wIdx+1][0]
+			}
+			f.Options = tfOptions
+		case "GEO":
+			f.Type = GeoField
 		case "NUMERIC":
 			f.Type = NumericField
 			nfOptions := NumericFieldOptions{}
 			f.Options = nfOptions
 			if sliceIndex(options, "SORTABLE") != -1 {
 				nfOptions.Sortable = true
 			}
+			f.Options = nfOptions
```

Comment:
@maguec why do you need this assigment twice? see line 514

Reply:
@gkorland you are correct I do not need it.  It has been removed